manifest_staging/charts/gatekeeper/crds/config-customresourcedefinition.yaml

```diff
@@ -24,14 +24,10 @@ spec:
       description: Config is the Schema for the configs API
       properties:
         apiVersion:
-          description: 'APIVersion defines the versioned schema of this representation
```

Comment:
It looks like this different formatting is due to kubebuilder version drift.

Reply:
s/kubebuilder/kustomize

Reply:
yes this is correctly the existing text wrap issue